src/providers/goDaddy.ts

```diff
@@ -2,6 +2,7 @@ import { sendRequest } from '../helpers/httpRequest'
 import { getProviderKeys } from '../lib/data'
 import { Provider, ServiceResponse } from '../types/general'
 import { ServiceKey } from '../types/admin'
+import fetch from 'node-fetch'
```

Comment:
sendRequest results in unexpected end of JSON input in the response body, but the response goes through and sets the CNAME records